java/opendcs/src/main/java/org/opendcs/database/impl/opendcs/OpenDcsOracleProvider.java

```diff
@@ -0,0 +1,246 @@
+package org.opendcs.database.impl.opendcs;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.statement.Call;
+import org.jdbi.v3.postgres.PostgresPlugin;
+import org.opendcs.spi.database.MigrationProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import decodes.dbimport.DbImport;
+import decodes.launcher.Profile;
+import decodes.tsdb.ImportComp;
+import decodes.tsdb.TimeSeriesDb;
+import decodes.util.DecodesSettings;
+import ilex.util.EnvExpander;
+import opendcs.opentsdb.OpenTsdb;
+
+/**
+ * OpenDCSPgProvider provides support for handling installation and updates of the OpenDCS-Postgres
```

Comment:
The comment here indicates Postgres, but the class name is Oracle.

I thought the CwmsOracleProvider took care of Oracle; but perhaps this is for HDB.

Reply:
This for the OpenDCS Oracle database (it's has one, it's used) I copied the PgProvider and forgot to update the comment.